packages/docs/src/page-configs/ui-elements/option-list/examples/WithComplexData.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <div>
-    <va-option-list
-      :options="options"
-      v-model="listValue"
-      valueBy="altValue"
-      disabledBy="altDisabled"
-      :textBy="option => option.altText"
-    />
-    Selected: <pre> {{ listValue }} </pre>
-  </div>
+  <va-option-list
+    :options="options"
+    v-model="listValue"
+    valueBy="altValue"
+    disabledBy="altDisabled"
```

Comment:
Aren't we using `kebab-case` notation for the components props?

Reply:
now, as I see it, we do not use this notation. But I think we need to do as recommended by the [vue style guide](https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing).
I think we should do it in a separate issue